src/pages/sponsors.js

```diff
@@ -0,0 +1,107 @@
+import React from 'react'
+import { Helmet } from "react-helmet"
+import styles from "../styles/sponsors.module.css"
+import { graphql, Link } from 'gatsby'
+
+const SponsorsPage = ( {data} ) => {
+
+	const getImageUrl = (companyName) => {
+		const URLRegex = /((([A-Za-z]{3,9}:(?:\/\/)?)(?:[\-;:&=\+\$,\w]+@)?[A-Za-z0-9\.\-]+|(?:www\.|[\-;:&=\+\$,\w]+@)[A-Za-z0-9\.\-]+)((?:\/[\+~%\/\.\w\-_]*)?\??(?:[\-\+=&;%@\.\w_]*)#?(?:[\.\!\/\\\w]*))?)/ ;
```

Comment:
Probably want to put url regexes into some sort of utility file as I'm sure this isn't the only time they will be needed. Have you seen https://github.com/kevva/url-regex. Might be nice to use that package as it's pretty robust url checks and not too much code. The tld package it relies on is kind of a big array but overall still a very small package.

Reply:
++